modules/alloy/src/demo/ts/ephox/alloy/demo/SliderDemo.ts

```diff
@@ -171,13 +171,13 @@ export default (): void => {
 
       onChange: (_slider, thumb, value: SliderValue) => {
         if (isValueX(value)) {
```

Comment:
This is no longer needed now that we're casting so it can be removed.

Reply:
And here I thought it was checking something useful, like only responding to X changes (there's another that only responds to Y). ok.